src/utilities/formatters.js

```diff
@@ -61,21 +61,7 @@ export const formatTimeDifference = duration => {
   );
 };
 
-export const getTickFormatter = () => {
-  let currentDay = null;
-  return tick => {
-    let formatted = '';
-    const nextCurrentDay = moment(tick).day();
-    if (currentDay !== nextCurrentDay) {
-      currentDay = moment(tick).day();
-      formatted = moment(tick).format('h:mma[\n](D/M)');
-    } else {
-      formatted = moment(tick).format('h:mma');
-    }
-
-    return formatted;
-  };
-};
+export const getTickFormatter = () => tick => moment(tick).format('h:mma[\n](D/M)');
```

Comment:
The screenshot is for the [`BreachChart`](https://github.com/openmsupply/mobile/blob/46b159479ad7a0e52a14658107e26ad0125c7a09/src/widgets/BreachChart.js#L60) which doesn't use this function (so doesn't fix the issue with that chart. Maybe we can just use this function in that component?

Personally I think it looks good if you don't display redundant dates, especially if you're viewing a small range

If we do keep it like this, I don't think there's any point to have it return a function given there's no closure

Reply:
Possibly, but given dates are only redundant when you've been logging for < 1 day, is it worth it for the potential resultant confusion?

Reply:
The only difference this change makes is that the date now shows when you're viewing logs on the same day. The original function would show a date if subsequent ticks were on different days and only when subsequent (am I using this right) ticks were on the same day would remove the day.

Reply:
> The only difference this change makes is that the date now shows on all logs.

That was the idea 😃. I think the conditional date rendering assumes that the user will understand that a time without a date implies the date is the current day.  As this data is quite mission critical, I would think including potentially redundant information is better than omitting it and assuming users will understand the implication.

TLDR; I might not have understood the argument for omitting the date for the current day, but AFAICS the UI looks fine either way, so I can't see a strong argument for adding potential ambiguity here?

But, if a user wanted it that way, I can fix to revert it back!